config/software/nokogiri.rb

```diff
@@ -67,6 +67,7 @@
     env["NOKOGIRI_USE_SYSTEM_LIBRARIES"] = "true"
 
     gem_command += [
+      "--platform ruby",
```

Comment:
Is this going to be an issue on windows?

Reply:
no check the if statement, we don't hit this section of code on windows because we do use the fat gem.

Reply:
https://github.com/chef/omnibus-software/blob/lcg/nokogiri-ruby-platform/config/software/nokogiri.rb#L25

Reply:
🤦‍♂